src/vellum/workflows/nodes/core/inline_subworkflow_node/node.py

```diff
@@ -140,3 +141,8 @@ def __annotate_outputs_class__(cls, outputs_class: Type[BaseOutputs], reference:
         # Subclasses of InlineSubworkflowNode can override this method to provider their own
         # approach to annotating the outputs class based on the `subworkflow.Outputs`
         setattr(outputs_class, reference.name, reference)
+
+        if not hasattr(cls, "__output_ids__"):
+            cls.__output_ids__ = {}
+
+        cls.__output_ids__[reference.name] = uuid4_from_hash(f"{cls.__id__}|{reference.name}")
```

Comment:
oh snap, this is bc at the moment that BaseNodeMeta runs, this output class isn't defined. Nice catch.